resources/lib/channels/fr/rmcbfmplay.py

```diff
@@ -25,15 +25,23 @@
                                       get_selected_item_label)
 from resources.lib.menu_utils import item_post_treatment
 
+
 @Route.register
 def get_token():
     addon = xbmcaddon.Addon("plugin.video.catchuptvandmore")
-    if addon.getSetting('rmcbfmplay.login') == '' or \addon.getSetting('rmcbfmplay.password') == '':
```

Comment:
@joaopa00  @Arias800 `or \addon.` resulted in invalid syntax, see the Flake8 error message by Flake8 in https://github.com/Catch-up-TV-and-More/plugin.video.catchuptvandmore/commit/7962169825662bc3538a1f1c40bd0664758a9bb4 :sweat_smile: 

No worries, I took the opportunity of resolving this merge conflict to also correct this bug and address introduced Flake8 violations so we are almost back to being able to rely on Flake8 checks in the CI :slightly_smiling_face::+1:

Reply:
I take note, I will be more careful next time.
Thanks for the information.

Reply:
> I take note, I will be more careful next time. Thanks for the information.

Really no worries @Arias800 , I'm kinda trying right now to tackle all Flake8 issues in this repo, then make a patch proposal to fix the CI on Flake8 checks, so it runs properly on PR branches, allowing contributors can see any issues ASAP instead of when PR has landed in `dev` (already too late...), and at the end everyone will be happier :hugs: :tada: